common/djangoapps/util/json_request.py

```diff
@@ -66,7 +66,7 @@ def __init__(self, resp_obj=None, status=None, encoder=EDXJSONEncoder,
         elif isinstance(resp_obj, QuerySet):
             content = serialize('json', resp_obj)
         else:
-            content = json.dumps(resp_obj, cls=encoder, indent=2, ensure_ascii=False)
+            content = json.dumps(resp_obj, cls=encoder, indent=2, ensure_ascii=True)
```

Comment:
@mushtaqak , why did you change this? Is it necessary? Will this affect other things?